### PR TITLE
Add serverless-bench image

### PR DESCRIPTION
**Pull request by moelsayed**
**Description**

Changes proposed in this pull request:

- Add serverless-bench image

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/12849

### CODE REVIEW

**Review by pPrecel — DISMISSED** [review]

**Review by skhalash — DISMISSED** [review]

File: serverless-bench/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Serverless-Bench
+
+This image is based on `locustio/locust` image. It also include a locust test configuration and a simple script to run the test and output the results in json to stdout. The logs are collected via a log sink and is pushed to google big query for further analysis.
```
Comment (mmitoraj) [inline serverless-bench/README.md:3]:
```suggestion
The Serverless-Bench image is based on the `locustio/locust` image. It also includes a locust test configuration and a simple script to run the test and output the results in JSON to stdout. The logs are collected via a log sink and are pushed to Google Cloud BigQuery for further analysis.  
```

**Review by mmitoraj — COMMENTED** [review]
I added one comment. Please have a look.

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**